### PR TITLE
Small edits

### PR DESCRIPTION
**Pull request by kmadk**

### CODE REVIEW

File: protocol/learn/overview.md [hunk 1]
```diff
@@ -42,10 +42,10 @@ The last core contract stores info about which surface contracts each project is
 
 There are currently 3 surface contracts that manage how projects manage funds and define how all core contracts should be used together. Anyone can write new surface contracts for projects to use.
 
-* [`JBController`](../specifications/contracts/or-controllers/jbcontroller/) stitches together funding cycles and community tokens, making sure all activity is accounted for and correct.
+* [`JBController`](../specifications/contracts/or-controllers/jbcontroller/) stitches together funding cycles and community tokens, allowing for curated control, accounting, and token management of specific projects environments.
```
Comment (mejango) [inline protocol/learn/overview.md]: can maybe leave out the last part "of specific projects environments" or reword.

File: protocol/learn/overview.md [hunk 2]
```diff
@@ -42,10 +42,10 @@ The last core contract stores info about which surface contracts each project is
...
 
-The `JBETHPaymentTerminal` inherits from the `IJBTerminal` interface. Projects are welcome to roll their own `IJBTerminal` to accept funds through. This can be useful to accept other tokens as payment, bypass protocol fees, or attempt some other funky design. A project can add/remove terminals using the [`JBDirectory.addTerminalOf(...)`](../specifications/contracts/jbdirectory/write/addterminalof.md) and [`JBDirectory.removeTerminalOf(...)`](../specifications/contracts/jbdirectory/write/removeterminalof.md).
+The `JBETHPaymentTerminal` inherits from the `IJBTerminal` interface. Projects are welcome to roll their own `IJBTerminal` to accept funds through. This can be useful to accept other tokens as payment, bypass protocol fees, or attempt some other funky design. A project can add/remove terminals of the Core [`JBDirectory`](../specifications/contracts/jbdirectory/) Contract using the [`JBDirectory.addTerminalOf(...)`](../specifications/contracts/jbdirectory/write/addterminalof.md) and [`JBDirectory.removeTerminalOf(...)`](../specifications/contracts/jbdirectory/write/removeterminalof.md) functions.
```
Comment (mejango) [inline protocol/learn/overview.md]: "from the Core" instead of "of the Core" ?

File: protocol/learn/topics/funding-cycle.md [hunk 3]
```diff
@@ -12,5 +12,5 @@
...
\ No newline at end of file
+* A funding cycle's `ballot` property is useful for setting rules according to which any proposed reconfiguration to subsequent cycles must adhere. This is useful for community oriented projects as it can prevent a project owner from malignantly updating an upcoming cycle's configuration moments before it begins without the broader community's consent. A funding cycle's ballot status, which is a [`JBBallotState`](../../specifications/enums/jbballotstate.md) enumeration, can be found by reading from [`JBFundingCycleStore.currentBallotStateOf(...)`](../../specifications/contracts/jbfundingcyclestore/read/currentballotstateof.md).
```
Comment (mejango) [inline protocol/learn/topics/funding-cycle.md]: maliciously?

Comment (mejango) [reply protocol/learn/topics/funding-cycle.md]: i suppose malignantly works too :)

File: protocol/learn/topics/project.md [hunk 4]
```diff
@@ -17,7 +17,7 @@
 
 For each project, the following functions can only be accessed by either the address that owns the project's NFT or by operator addresses explicitly allowed by the address that owns the project's NFT. Operators are only valid in the context of a particular owner – if the NFT changes hands, the operators for the project must be set again by the new owner.
 
-An address can set operators for its project through the [`JBOperatorStore.setOperator(..`](../../specifications/contracts/jboperatorstore/write/setoperator.md)`.)` transaction, using the indexes from the [`JBOperations`](../../specifications/libraries/jboperations.md) library.
+An address can set operators for its project through the [`JBOperatorStore.setOperator(..`](../../specifications/contracts/jboperatorstore/write/setoperator.md)`.)` transaction, using the indexes from the [`JBOperations`](../../specifications/libraries/jboperations.md) library. The specific permissions a particular Operator is allowed depend on the specific parameters the admin allows them. The admin's permissions include calls to all of the following functions whereas the permissions of a specific Operator may vary.
```
Comment (mejango) [inline protocol/learn/topics/project.md]: Doesn't read succinctly imo.

Maybe: ".... Operator is allowed are granted by the admin explicitly. Each of the following functions can be called by the admin, and also by any operator that has been granted permission to call the function by the admin."

**Review by mejango — CHANGES_REQUESTED** [review]
A few suggestions for wording changes, but in general this is a great upgrade. thanks!

File: protocol/learn/overview.md (hunk 1 above)
Comment (kmadk) [reply protocol/learn/overview.md]:
```suggestion
* [`JBController`](../specifications/contracts/or-controllers/jbcontroller/) stitches together funding cycles and community tokens, allowing for curated control, accounting, and token management.
```

**Pull request merged**